internal/pkg/message/outbox.go

```diff
@@ -88,8 +88,8 @@ func (ob *Outbox) Send(ctx context.Context, from, to address.Address, value type
 			msgSendErrCt.Inc(ctx, 1)
 		}
 		ob.journal.Write("Send",
-			"to", to.String(), "from", from.String(), "value", value.AsBigInt().Uint64(), "method", method,
-			"gasPrice", gasPrice.AsBigInt().Uint64(), "gasLimit", uint64(gasLimit), "bcast", bcast,
+			"to", to.String(), "from", from.String(), "value", value.Int.Uint64(), "method", method,
```

Comment:
Would welcome a stringification method on big.Int in specs-actors

Reply:
Good deal.  I'll file an issue with this and other proposals for shared stuff that should happen on that type (name change, FIL to AttoFIL conversion)